draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -382,7 +382,7 @@ Disclosures are created differently depending on whether a claim is an object pr
 For each claim that is an object property and that is to be made selectively disclosable, the Issuer MUST create a Disclosure as follows:
 
  * Create a JSON array of three elements in this order:
-   1. A salt value. MUST be a string. See (#salt-entropy) for security considerations. It is RECOMMENDED to base64url-encode a minimum of 128 bits of cryptographically secure random data, producing a string. The salt value MUST be unique for each claim that is to be selectively disclosed. The Issuer MUST NOT reveal the salt value to any party other than the Holder.
+   1. A salt value. MUST be a string. See (#salt-entropy) for security considerations. To achieve recommended entropy of the salt, Issuer is RECOMMENDED to base64url-encode a minimum of 128 bits of cryptographically secure random data, producing a string. The salt value MUST be unique for each claim that is to be selectively disclosed. The Issuer MUST NOT reveal the salt value to any party other than the Holder.
```

Comment:
The RECOMMENDED always felt too strong and a little out of place. And the minimum of feels unneeded and redundant to what's in (#salt-entropy)
```suggestion
   1. A salt value. MUST be a string. See (#salt-entropy) for security considerations. To achieve recommended entropy of the salt, the Issuer can base64url-encode 128 bits of cryptographically secure random data, producing a string. The salt value MUST be unique for each claim that is to be selectively disclosed. The Issuer MUST NOT reveal the salt value to any party other than the Holder.
```

Reply:
Looks good to me, but I'd throw in a definite article.

```suggestion
1. A salt value. MUST be a string. See (#salt-entropy) for security considerations. To achieve the recommended entropy of the salt, the Issuer can base64url-encode 128 bits of cryptographically secure random data, producing a string. The salt value MUST be unique for each claim that is to be selectively disclosed. The Issuer MUST NOT reveal the salt value to any party other than the Holder.
```